deployment/media-insights-stack.yaml

```diff
@@ -60,6 +59,9 @@ Mappings:
       CodeKeyPrefix: "media_insights_engine/%%VERSION%%"
       TemplateKeyPrefix: "media_insights_engine/%%VERSION%%"
       FrameworkVersion: "%%VERSION%%"
+  AnonymousData:
+    SendAnonymousData:
+      Data: Yes
```

Comment:
Should this be a user parameter or a mapping?

Reply:
It is intended to be something that users can opt out of, so yes, a parameter.

Reply:
This option is now a user configurable CF parameter, so I'm resolving this conversation now.